orchestra/examples/duo.rs

```diff
@@ -63,7 +63,7 @@ impl<Context> Fortified {
 
 #[orchestra(signal=SigSigSig, event=EvX, error=Yikes, gen=AllMessages)]
 struct Duo<T, U, V, W> {
-	#[subsystem(consumes: MsgStrukt, sends: [Plinko], message_capacity: 32768)]
+	#[subsystem(consumes: MsgStrukt, sends: [Plinko], message_capacity: 32768, signal_capacity: 128)]
```

Comment:
I would add test for the overrides(ensure we block when channel full) to ensure it doesn't break over time.